src/run_loop.ml

```diff
@@ -0,0 +1,19 @@
+open Tsdl
+
+let rec event_loop () =
+  match Sdl.poll_event () with
+  | Some event ->
+      (match Sdl.Event.(enum (get event typ)) with
+      | `Quit -> ()
+      | _ -> event_loop ())
+  | None -> event_loop ()
+
+let run () =
```

Comment:
To expand on @patricoferris's comment - these functions aren't needed. What you need to do is modify the current run loop in Claudius in `base.ml` to ensure that if no changes have been made from the tick function Claudius calls that the user of Claudius provided, then it doesn't do any of the work to re-draw the screen.